classes/teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization, age, name)
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same Here, As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Constructor%20with%20name%2C%20age%2C%20and%20parent_permission%20as%20parameter.%20name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true.), Please add `parent_permission` as parameter and ensure that `name` and `parent_permission` have default values of "Unknown" and true.